### PR TITLE
{tools}[fosscuda/2019b] torchtext v0.5.0 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
depends on ~~#10192~~

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node372.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/b3c3859cc3f0bc61c8126d725ccaebb3 for a full test report.

**Comment by smoors** [comment]
closing and reopening to trigger tests

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/f4ca2fa1b207b437eeae3de0348183ab for a full test report.

File: easybuild/easyconfigs/t/torchtext/torchtext-0.5.0-fosscuda-2019b-PyTorch-1.4.0--Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    ('tqdm', '4.41.1'),
+    ('PyTorch', local_pytorchver, local_pyversuff),
+]
+
+use_pip = True
+download_dep_fail = True
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages/torchtext'],
```
Comment (Micket) [inline easybuild/easyconfigs/t/torchtext/torchtext-0.5.0-fosscuda-2019b-PyTorch-1.4.0--Python-3.7.4.eb]: Isn't this the default in PythonPackage already?

Comment (smoors) [reply easybuild/easyconfigs/t/torchtext/torchtext-0.5.0-fosscuda-2019b-PyTorch-1.4.0--Python-3.7.4.eb]: ok, removed.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
hebbe-c1 - Linux centos linux 7.7.1908, Intel Core Processor (Haswell, no TSX), Python 2.7.5
See https://gist.github.com/e48ecdca6c2c1b6fb1d2c5dca4d930ac for a full test report.

**Comment by Micket** [comment]
Going in, thanks @smoors!

**Pull request merged**